scripts/update.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+if [ "${UPDATE_ON_BOOT}" = false ]; then
+    echo "Update on Boot needs to be enabled for auto updating"
+    exit 1
+fi
+
+CURRENTBUILD=$(awk '/buildid/{ print $2 }' < palworld/steamapps/appmanifest_2394010.acf)
+TARGETBUILD=$(curl https://api.steamcmd.net/v1/info/2394010 --silent | grep -P '"public": {"buildid": "\d+"' -o | sed -r 's/.*("[0-9]+")$/\1/')
+
+if [ "$CURRENTBUILD" != "$TARGETBUILD" ]; then
+    echo "New Build was found. Updating the server from $CURRENTBUILD to $TARGETBUILD."
+    if [ "${RCON_ENABLED}" = true ]; then
+        rm /palworld/steamapps/appmanifest_2394010.acf
+        rcon-cli -c /home/steam/server/rcon.yaml "broadcast The_Server_will_update_in_${WARN_MINUTES}_Minutes"
+        sleep "${WARN_MINUTES}m"
+        rcon-cli -c /home/steam/server/rcon.yaml save
```

Comment:
Probably a good idea to run a backup before the update.
```suggestion
        backup
```